feat: Update description.txt (#468)
openfoodfacts/fastlane-descriptions-smoothie: fastlane/ios/fastlane/metadata/en-US/description.txt

Objective Analysis: Product evaluations and information are presented objectively. No brand can influence our ratings.
No Advertising: Our focus is transparency, not profit. You won't find ads in our app.
Community Driven: Built and maintained by tens of thousands of volunteers worldwide. We are made by everyone, for everyone.
◆ INSTANT INSIGHTS, PERSONALIZED FOR YOU ◆

Get clear, color-coded results (🟢 Green: matches your criteria, 🔴 Red: concern, ⚫ Gray: more info needed) tailored to your preferences:

FOR YOUR HEALTH:
Nutri-Score (A-E): Nutritional quality at a glance.
NOVA Group (1-4): Identify and avoid ultra-processed foods.
Additives: Understand their presence and risk levels based on EFSA reviews.
Nutrients: Track salt, sugar, fat, and more.
FOR THE PLANET:
Eco-Score (A-E): Assess the environmental footprint (CO2, packaging, origins).
FOR YOUR NEEDS:
Allergen Alerts: Set up warnings for Milk, Gluten, Eggs, Soy, Nuts, Fish, Celery, Mustard, Sulphites, Peanuts, Sesame, Crustaceans, Molluscs, Lupin.
Dietary Filters: Vegan, vegetarian, organic, halal, kosher, etc.
◆ COMPARE & DISCOVER ◆

> Easily compare products side-by-side on the shelf. While we don't directly recommend alternatives like some apps, we empower YOU to find healthier and more sustainable options by providing comprehensive, unbiased data.

Author: ```suggestion
Easily compare products side-by-side on the shelf. We can provide alternatives for problematic products, BUT WITH YOUR OWN CRITERION.
```
